keras_nlp/src/models/llama/llama_presets.py

```diff
@@ -25,6 +25,16 @@
         },
         "kaggle_handle": "kaggle://keras/llama2/keras/llama2_7b_en/1",
     },
+    "llama2_7b_en_int8": {
+        "metadata": {
+            "description": "LLaMA 2 7B Quantized Base model",
```

Comment:
Maybe `LLaMA 2 7B base model with weight quantized to int8.`

Reply:
I have updated the description to follow Gemma's convention.

Additionally, I used `"with activation and weights quantized to int8."` to indicate that we are using dynamic int8 quantization instead of weights-only quantization.

WDYT?

Reply:
sgtm!